src/components/ui/MicVolumeIndicator/index.tsx

```diff
@@ -27,10 +27,10 @@ export const MicVolumeIndicator = forwardRef(
 
     return (
       <StyledMicVolumeIndicator
-        {...rest}
```

Comment:
Hmm, we might want to pull out classNames prop and combine it with `ch-mic-volume-indicator` explicitly

Reply:
okok yeah that makes sense. Good call out! So you mean this right?
```
className={`${className} ch-mic-volume-indicator`}
```

Reply:
You'll want to explicitly check if className is defined, otherwise you'll get `undefined ch-mic-volume-indecator`. But other than that yep looks good

Reply:
Yes, will check that. Thanks!